### PR TITLE
Skillonomy 75 translate

### PR DESCRIPTION
**Pull request by RuslanSkiraRaccoongang**
**Description:** Translation of the text from English in to Russian

**Youtrack:** [SKILLONOMY-75](https://youtrack.raccoongang.com/issue/SKILLONOMY-75)
[SKILLONOMY-81](https://youtrack.raccoongang.com/issue/SKILLONOMY-81)

### CODE REVIEW

File: conf/locale/ru/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -8412,7 +8417,11 @@ msgstr "или создать новую в этом сервисе"
 
 #: lms/templates/student_account/register.underscore
 msgid "Create account"
-msgstr ""
+msgstr "Coздать профиль"
+
+#: lms/templates/student_account/register.underscore
+msgid "Already have an {platformName} account?"
```
Comment (OPersian) [inline conf/locale/ru/LC_MESSAGES/djangojs.po]: @RuslanSkiraRaccoongang please push respective changes in the`register.underscore` with a new string marked for translation

Comment (OPersian) [reply conf/locale/ru/LC_MESSAGES/djangojs.po]: Also, let's probably not wait for the platform config to arrive.
Once any changes happen in strings, I'll update po files

File: conf/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -15798,11 +15798,18 @@ msgid ""
...
+"MAMA - это обучение от ведущих преподавателей и высокое качество полученных знаний онлайн."
+"Жми на ссылку:"
 
 #: lms/templates/emails/activation_email.txt
 msgid "Enjoy learning with {platform_name}."
 msgstr ""
-
+"Важно:"
+"Вы не сможете снова войти в свою учетную запись, пока не активируете ее."
+"Если Вам понадобится помощь, пишите нам на электронною почту academy@mama.mobytize.com"
```
Comment (oleksandra-martyntseva) [inline conf/locale/ru/LC_MESSAGES/django.po]: Is this email really supposed to be hardcoded here?

File: common/djangoapps/terrain/steps.py [hunk 3]
```diff
@@ -230,8 +230,6 @@ def i_answer_prompts_with(step, prompt):
 @step('I run ipdb')
 def run_ipdb(_step):
     """Run ipdb as step for easy debugging"""
-    import ipdb
```
Comment (oleksandra-martyntseva) [inline common/djangoapps/terrain/steps.py]: Looks like this was the sole purpose of this method, why did you remove the breakpoint?

File: conf/locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -15798,11 +15798,18 @@ msgid ""
 "engaging, high-quality {platform_name} courses. Note that you will not be "
 "able to log back into your account until you have activated it."
 msgstr ""
+"Привет!"
+"Используйте нашу ссылку, чтобы активировать свою учетную запись и получить доступ к курсам MAMA."
```
Comment (oleksandra-martyntseva) [inline conf/locale/ru/LC_MESSAGES/django.po]: Is "MAMA" supposed to be hardcoded? I see that the english version uses {platform_name} instead. Shouldn't the russian version also use some variable?

File: conf/locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -12454,7 +12454,7 @@ msgid ""
 msgstr ""
 "{begin_strong}Предупреждение:{end_strong} ваш браузер поддерживается не "
 "полностью. Настоятельно рекомендуем использовать {chrome_link} или "
-"{ff_link}."
+"{ff_link}."
```
Comment (oleksandra-martyntseva) [inline conf/locale/ru/LC_MESSAGES/django.po]: Maybe remove these added spaces?

File: conf/locale/ru/LC_MESSAGES/django.po (hunk 2 above)
Comment (RuslanSkiraRaccoongang) [reply conf/locale/ru/LC_MESSAGES/django.po]: I didn't find any variable for using instead. And thats why I hardcoded this here.

File: conf/locale/ru/LC_MESSAGES/django.po (hunk 4 above)
Comment (RuslanSkiraRaccoongang) [reply conf/locale/ru/LC_MESSAGES/django.po]: Fixed

File: common/djangoapps/terrain/steps.py (hunk 3 above)
Comment (RuslanSkiraRaccoongang) [reply common/djangoapps/terrain/steps.py]: Return.

File: conf/locale/ru/LC_MESSAGES/django.po (hunk 5 above)
Comment (RuslanSkiraRaccoongang) [reply conf/locale/ru/LC_MESSAGES/django.po]: Done

File: conf/locale/ru/LC_MESSAGES/django.po (hunk 2 above)
Comment (oleksandra-martyntseva) [reply conf/locale/ru/LC_MESSAGES/django.po]: There is an english line "If you need help, please use our web form at {support_url} or email " - isn't it the same?

File: conf/locale/ru/LC_MESSAGES/djangojs.po [hunk 6]
```diff
@@ -4797,6 +4800,8 @@ msgid ""
 " password. Didn't receive the message? Contact {anchorStart}technical "
 "support{anchorEnd}."
 msgstr ""
+"Мы отправили Вам сообщение на {email}. Нажмите на ссылку в сообщении, чтобы сбросить пароль. Не получили сообщение?"
+"Обратитесь в нашу службу технической поддержки: {anchorStart}academy@mama.mobytize.com{anchorEnd}."
```
Comment (OPersian) [inline conf/locale/ru/LC_MESSAGES/djangojs.po]: Hardcoding here as well

Comment (OPersian) [reply conf/locale/ru/LC_MESSAGES/djangojs.po]: @RuslanSkiraRaccoongang please consider using `{support_email}` here

File: conf/locale/ru/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (RuslanSkiraRaccoongang) [reply conf/locale/ru/LC_MESSAGES/djangojs.po]: image

Comment (RuslanSkiraRaccoongang) [reply conf/locale/ru/LC_MESSAGES/djangojs.po]: Need to always add this data
